allennlp/tests/training/metrics/fbeta_measure_test.py

```diff
@@ -77,17 +87,21 @@ def test_fbeta_multiclass_metric(self):
         fscores = metric["fscore"]
 
         # check value
-        numpy.testing.assert_almost_equal(precisions, self.desired_precisions, decimal=2)
-        numpy.testing.assert_almost_equal(recalls, self.desired_recalls, decimal=2)
-        numpy.testing.assert_almost_equal(fscores, self.desired_fscores, decimal=2)
+        assert_allclose(precisions, self.desired_precisions)
+        assert_allclose(recalls, self.desired_recalls)
+        assert_allclose(fscores, self.desired_fscores)
```

Comment:
In many of these changes, the expected precision is actually stronger. It works because I changed `0.33` as the expected value to `1 / 3`.